miniapp/USERGUIDE.md

```diff
@@ -623,6 +623,50 @@ mini app scheme and should close external webview.
 
 Using `#ExternalResultHandler.emitResult(String)` to transmit the url string to mini app view.
 
+### File choosing
+**API Docs:** [MiniAppFileChooser](api/com.rakuten.tech.mobile.miniapp.file/-mini-app-file-chooser/)
+
+The mini app is able to choose the file which is requested using HTML forms with 'file' input type whenever users press a "Select file" button.
+HostApp can use `MiniAppFileChooser` to override `onShowFileChooser` to customize file choosing mode and other options.
```

Comment:
I think these examples are in the wrong order: Most of the time the Host App will want to use `MiniAppFileChooserDefault`. So we should put that example first and make sure it's clear that it's the recommended usage.